packages/ui/ui/package.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
It's a little weird  that this is in packages/ui/ui but called slate-plugins-ui since ui also contains the toolbar package... maybe it should be something like packages/ui/styled-components and slate-plugins-ui-styled-components or something?

Reply:
Indeed, I'll rename it to `slate-plugins-styled-components` (`-ui` is not necessary)